tools/csv-generator/csv-generator.go

```diff
@@ -63,7 +64,8 @@ func main() {
 	virtExportProxyImage := flag.String("virt-export-proxy-image", "", "custom image for virt-export-proxy. "+customImageExample)
 	virtExportServerImage := flag.String("virt-export-server-image", "", "custom image for virt-export-server. "+customImageExample)
 	gsImage := flag.String("gs-image", "", "custom image for gs. "+customImageExample)
-	prHelperImage := flag.String("pr-helper-image", "", "custom image for gs. "+customImageExample)
+	prHelperImage := flag.String("pr-helper-image", "", "custom image for pr-helper. "+customImageExample)
```

Comment:
nit: this fix should probably be on its own commit

Reply:
Done. PTAL.

----
If you don't mind, how would you go about doing this? I'll share how I did it.
1. `git rebsae -i HEAD~4`
2. `edit` the specific commit - 6e083f4345 in this case.
3. Modify the line in question to what it was earlier.
4. Amend the commit with `git add . && git commit --amend`, and finish rebase with `git rebsae --continue`.
5. Make a new commit - b02c45b5 in this case.

I was hoping to be able to use `git reset` to do this instead, but couldn't figure it out.

Reply:
For removing something from a commit and moving it to another, I usually do the following:
1. `git rebase -i ...`
2. `edit` the specific commit
3. `undo` the commit
4. open and keep your editor open!  remove what you don't want to see
5. git add the changed buffers and `redo` the commit
6. on the editor, `ctrl-z` to get the changes back
7. git add && git commit the new commit
8. finish rebase

Now, the `undo` and `redo` are actually git alias that I have here, they are:
```
[alias]
    undo = reset --soft HEAD~
    redo = commit -c ORIG_HEAD
```

When you do `redo`, you get back the commit log from the commit you did `undo`

Pretty sure there are plenty ways to go around this, the important part is being able to do it :smile: 
Thanks again!